File: app/src/main/java/example/com/archtest/data/RepoInfo.java
package example.com.archtest.data;


import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.Required;

public class RepoInfo extends RealmObject {
	@PrimaryKey
	private int id;
	@Required
	private String name;

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}
}

File: app/src/main/java/example/com/archtest/data/RepoInfo.kt
package example.com.archtest.data

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import io.realm.annotations.Required

open class RepoInfo : RealmObject() {
    @PrimaryKey
    var id: Int = 0
    @Required
    var name: String? = null
}